app/src/components/widgets/charts/investigatedTrendChart/statusPageModeConfig.jsx

```diff
@@ -0,0 +1,142 @@
+import ReactDOMServer from 'react-dom/server';
+import { getItemColor } from 'components/widgets/charts/common/utils';
+import { TimelineTooltip } from './timelineTooltip';
+import { MESSAGES } from './common/constants';
+
+export const mapMinListLength = {
+  '1M': 30,
```

Comment:
Seems that this constants already used in the `pages/admin/projectStatusPage/constants.js` (`PERIOD_VALUES`).
Please move it upper (f.e. to `common/constants/statusPeriodValues.js`) and use everywhere across the PR